src/benchmarks/real-world/Roslyn/CompilerBenchmarks.csproj

```diff
@@ -6,6 +6,10 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <IsShipping>false</IsShipping>
+    <!-- this app is using internal Roslyn API and needs to be signed -->
+    <SignAssembly>true</SignAssembly>
+    <!-- ignore warning about BenchmarkDotNet.Extensions being not signed -->
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
This will not discriminate other warnings in the code. Is there a more specific way to do this for the extensions only?

Reply:
good point, I have changed it to ignore only this particular warning

```xml
<NoWarn>$(NoWarn);8002</NoWarn>
```